src/type-checker.cc

```diff
@@ -366,6 +366,15 @@ Result TypeChecker::OnAtomicWait(Opcode opcode) {
   return CheckOpcode3(opcode);
 }
 
+Result TypeChecker::OnAtomicFence(uint32_t consistency_model) {
+  if (consistency_model != 0x0) {
+    PrintError("bad atomic.fence consistency model: %u",
+               consistency_model);
+    return Result::Error;
+  }
+  return Result::Ok;
+}
```

Comment:
Is the type checker the right place for this?  i.e. is this a more general validation thing than a type checking thing?